src/rich-text/commands.ts

```diff
@@ -558,20 +558,27 @@ export function insertLinkCommand(
  * Creates an `active` method that returns true of the current selection is/contained in the current block type
  * @param nodeType The type of the node to check for
  */
-function nodeTypeActive(nodeType: NodeType) {
+function nodeTypeActive(
+    nodeType: NodeType,
+    attrs?: { [key: string]: unknown }
+) {
     return function (state: EditorState) {
         const { from, to } = state.selection;
         let isNodeType = false;
+        let hasAttrs = attrs ? false : true;
```

Comment:
nit: There's a more "JS-y" way of doing this:

```suggestion
        let hasAttrs = !attrs;
```

Additionally, I'd rename this to `passesAttrsCheck` or something similar. `hasAttrs` runs a bit counter to what the code is actually doing, which is checking that all attributes passed are checked and match the attrs on the node.